CRM/Activity/Selector/Search.php

```diff
@@ -291,10 +291,10 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL) {
         $accessMailingReport = TRUE;
       }
       $activityActions = new CRM_Activity_Selector_Activity($result->contact_id, NULL);
-      $actionLinks = $activityActions->actionLinks($activityTypeId,
-        CRM_Utils_Array::value('source_record_id', $row),
+      $actionLinks = $activityActions::actionLinks($activityTypeId,
```

Comment:
The only thing is why $activityActions even needs to be instantiated as a variable but not a blocker.

Reply:
Oh right - yeah - I didn't look at that - the change on this line was just to get rid of `CRM_Utils_Array::value`